CHANGELOG.md

```diff
@@ -1,6 +1,13 @@
 # Change Log
 All notable changes to this project will be documented in this file.
 
+## Wazuh Docker v4.0.4_1.11.0
+
+### Added
+
+- Update to Wazuh version 4.0.4
```

Comment:
Hey @xr09,

Could link here the `wazuh/wazuh` `v4.0.4` changelog? https://github.com/wazuh/wazuh-ansible/blob/v4.0.4/CHANGELOG.md

Reply:
Fixed in 3a5500e